packages/create-cloudflare/turbo.json

```diff
@@ -1,11 +1,9 @@
 {
 	"$schema": "http://turbo.build/schema.json",
 	"extends": ["//"],
-
 	"pipeline": {
 		"build": {
 			"env": [
-				"VITEST",
```

Comment:
removing this after having run `pnpm run check:lint` and confirming that we don't use that anywhere in `create-cloudflare`